packages/@react-spectrum/toast/stories/Toast.stories.tsx

```diff
@@ -333,3 +334,37 @@ function MainLandmark(props) {
   let {landmarkProps} = useLandmark({...props, role: 'main'}, ref);
   return <main aria-label="Danni's unicorn corral" ref={ref} {...props} {...landmarkProps} style={{padding: 40, background: 'white'}}>{props.children}</main>;
 }
+
+export const withFullscreen = {
+  render: () => <FullscreenApp />,
+  parameters: {
+    disableToastContainer: true
+  }
+};
+
+function FullscreenApp(props) {
+  let ref = useRef(null);
+  let [isFullscreen, setFullscreen] = useState(false);
+  let fullscreenPress = () => {
+    if (!isFullscreen) {
+      ref.current.requestFullscreen();
+    }
+  };
+  useEffect(() => {
+    let onFullscreenChange = () => {
+      setFullscreen(!!document.fullscreenElement);
+    };
+    document.addEventListener('fullscreenchange', onFullscreenChange);
+    return () => document.removeEventListener('fullscreenchange', onFullscreenChange);
+  }, []);
+  return (
+    <div ref={ref} style={{position: 'fixed', top: 0, left: 0, width: '100vw', height: '100vh', backgroundColor: 'white'}}>
+      <UNSTABLE_PortalProvider getContainer={() => ref.current}>
```

Comment:
I know the iframe story works, but do we need a test of UNSTABLE_PortalProvider and iframe together?

Reply:
I don't follow the question. I think you're asking about iframes such as has been done for focus scope? I'm not going to worry about that use case right now.

Reply:
I'm asking if we need to test using iframes and UNSTABLE_PortalProvider at the same time. An example could behaving toast triggering from an iframe, but the page with the iframe and ToastContainer is using UNSTABLE_PortalProvider. Or some combination of the two like this which would add complexity and could break.